rosbag2_transport/test/resources/recorder_node_params.yaml

```diff
@@ -0,0 +1,33 @@
+recorder_params_node:
+  ros__parameters:
+    all: true
+    is_discovery_disabled: true
+    topics: ["topic", "other_topic"]
+    rmw_serialization_format: "cdr"
+    topic_polling_interval:
+      sec: 0
+      nsec: 10000000
+    regex: "[xyz]/topic"
+    exclude: "*"
+    node_prefix: "prefix"
+    compression_mode: "stream"
+    compression_format: "h264"
+    compression_queue_size: 10
+    compression_threads: 2
+    qos_profile_overrides_path: ""
+    include_hidden_topics: true
+    include_unpublished_topics: true
+    ignore_leaf_topics: false
+    start_paused: false
+    use_sim_time: false
+
+    storage_id: sqlite3
+    storage_config_uri: ""
+    max_bagfile_size: 12345
+    max_bagfile_duration: 54321
+    max_cache_size: 9898
+    storage_preset_profile: "none"
```

Comment:
@roncapat It looks like it would be more user friendly to separate storage and recorder parameters to sub-yaml nodes with respective names. 
However, not sure how it will look like in the code. Thoughts?